blank/skeleton/.github/workflows/extended-test.yaml

```diff
@@ -17,6 +17,6 @@ jobs:
 
   extendedtests:
     needs: [get-latest-version]
-    uses: coreeng/p2p/.github/workflows/p2p-workflow-extended-test.yaml@v1
+    uses: coreeng/p2p/.github/workflows/p2p-workflow-extended-test.yaml@v1.6.45
```

Comment:
Any reason to set this version? I'd just leave v1, that one is always updated to the latest

Reply:
done, had some issues initially when I was testing this but now all seems fine